src/app/components/Student.js

```diff
@@ -1,5 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import FormControl from '@material-ui/core/FormControl';
+import TextField from '@material-ui/core/TextField';
+import DateFnsUtils from '@date-io/date-fns';
+import { DatePicker, MuiPickersUtilsProvider } from 'material-ui-pickers';
+import EventIcon from '@material-ui/icons/Event';
+import Button from '@material-ui/core/Button';
 
-const Student = () => <h1>Student</h1>;
+const Student = () => {
+  const [selectedDate, handleDateChange] = useState(new Date());
+  const [name, handleName] = useState('');
```

Comment:
i love it!!! :anchor:

Reply:
OOOooo me too! Never used hooks before

Reply:
It's gone when redux form is in lol. All states are handled!